src/FixedDataTableCellGroup.js

```diff
@@ -193,11 +193,17 @@ class FixedDataTableCellGroup extends React.Component {
   }
 
   shouldComponentUpdate(/*object*/ nextProps) /*boolean*/ {
-    return (
-      !nextProps.isScrolling ||
-      this.props.rowIndex !== nextProps.rowIndex ||
-      this.props.left !== nextProps.left
-    );
+    /// if offsets haven't changed for the same row while scrolling, then skip update
+    if (nextProps.isScrolling) {
```

Comment:
Also check the offset. I can think of case where the offset changed due to change in columns width but being lost. Costs little to nothing in performance.